biolink-model.yaml

```diff
@@ -1878,7 +1878,7 @@ slots:
     mappings:
       - pav:createdWith
 
-  downloadURL:
+  download url:
```

Comment:
note this will render downloadUrl. Is this OK?

You can always override with a specific `slot_uri: biolink:downloadURL`

Reply:
Since this is a slot, I thought it would render "download_url"?

Reply:
Yes, this will render as `download_url`